src/nvim/ex_docmd.c

```diff
@@ -9411,3 +9411,36 @@ static void ex_terminal(exarg_T *eap)
     xfree(name);
   }
 }
+
+static void ex_plug(exarg_T *eap)
+{
+  char *path = NULL;
+  if (*eap->arg != NUL) {
+    if (os_isdir(eap->arg)) {  // full path: plug ~/.config/nvim/plugged/coolplugin
+      path = (char *)vim_strsave_fnameescape(eap->arg, FALSE);
+    } else {  // spec: plug neovim/neovimplugin
+      char_u *user_vim = vim_strsave(p_rtp);
+      strtok((char *)user_vim, ","); // split(&runtimepath, ',')[0]
+      char_u *basename = path_tail((char_u *)eap->arg); // 'neovim/neovimplugin' -> 'neovimplugin'
```

Comment:
@fmoralesc Completely invalid, at least for VAM. Plugin name like `github:ludovicchabant/vim-gutentags` is translated to `github-ludovicchabant-vim-gutentags` without requiring user to specify anything. Just `vim-gutentags` is not used because with some common names this name may conflict with plugins found in the database.